worksheet_A/TerminalHacking.cpp

```diff
@@ -3,9 +3,32 @@
 
 #include "stdafx.h"
 #include "WordList.h"
+#include <string>
+#include <iostream>
+
+
+using namespace std;
 
 const int wordLength = 5;
 const int numberOfWords = 15;
+int PlayerLives = 4;
+int Likeness = 0;
+string Guess = "";
+
+void HackingIntro();
```

Comment:
Debatable whether this is better than simply putting the function body here, particularly since `GetLikeness` is defined directly below.